tensorflow-examples/src/main/java/org/tensorflow/model/examples/objectdetection/FasterRcnnInception.java

```diff
@@ -0,0 +1,302 @@
+package org.tensorflow.model.examples.objectdetection;
```

Comment:
Can you add the copyright/license statement?

Reply:
Will do.